app/src/main/java/com/sopt/androidstudy/presentation/compose/ComponentActivity.kt

```diff
@@ -0,0 +1,203 @@
+            LocalDateTime.of(2022, 11, 8, 15, 0)
+        ),
+
+        ToDoTask(
+            ToDoType.EXERCISE,
+            "데드리프트 30회 5세트",
+            LocalDateTime.of(2022, 11, 25, 7, 0)
+        ),
+        ToDoTask(
+            ToDoType.EXERCISE,
+            "벤치프레스 30회 5세트",
+            LocalDateTime.of(2022, 11, 25, 8, 0)
+        ),
+        ToDoTask(
+            ToDoType.STUDY,
+            "토스 자소서 쓰기",
+            LocalDateTime.of(2022, 11, 25, 16, 0)
+        ),
+        ToDoTask(
+            ToDoType.MEETING,
+            "위시보드 회의", // 하우스, 뮤멘트, 노바로 커스텀 하세요!
+            LocalDateTime.of(2022, 11, 25, 20, 0)
+        ),
+        ToDoTask(
+            ToDoType.STUDY,
+            "심화스터디",
+            LocalDateTime.of(2022, 11, 25, 22, 0)
+        ),
+        ToDoTask(
+            ToDoType.EXERCISE,
+            "풋살하기",
+            LocalDateTime.of(2022, 12, 8, 15, 0)
+        ),
+        ToDoTask(
+            ToDoType.STUDY,
+            "코테 뿌시기",
+            LocalDateTime.of(2022, 12, 8, 20, 0)
+        ),
+        ToDoTask(
+            ToDoType.HANG_OUT,
+            "제주도 여행",
+            LocalDateTime.of(2022, 12, 18, 15, 0)
+        ),
+
+        ToDoTask(
+            ToDoType.STUDY,
+            "밀린 강의 청산",
+            LocalDateTime.of(2022, 12, 27, 20, 0)
+        ),
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            AndroidSturdyTheme {
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colors.background
+                ) {
+                    val selectDate = remember {
+                        mutableStateOf<Int?>(null)
+                    }
+                    val currentMonth = remember {
+                        mutableStateOf(LocalDateTime.now().month.value)
+                    }
+
+                    Column() {
+                        Row(
+                            horizontalArrangement = Arrangement.SpaceEvenly, modifier = Modifier
+                                .padding(0.dp, 12.dp, 0.dp, 20.dp)
+                                .fillMaxWidth()
+                        ) {
+                            Text(
+                                text = "이전",
+                                fontSize = 20.sp,
+                                color = Color.Black,
+                                modifier = Modifier
+                                    .border(
+                                        width = 1.dp,
+                                        color = Color.Gray,
+                                        shape = RectangleShape
+                                    )
+                                    .padding(10.dp)
+                                    .clickable {
+                                        selectDate.value = null
+                                        currentMonth.value = currentMonth.value.dec()
```

Comment:
```suggestion
                           currentMonth.value = currentMonth.value.dec().takeIf { it >= 1 } ?: currentMonth.value
```
1월에서 뒤로가기를 눌렀을 경우 앱 버그가 있습니당
위와 같이 고쳐주세용

Reply:
이것도 12월과 동일! 오케이요!